Dockerfile

```diff
@@ -46,14 +46,17 @@ FROM build AS dashboard-stage
 
 WORKDIR /dashboard
 
+## Install stage dependencies
+COPY dashboard/package.json dashboard/yarn.lock ./
+RUN yarn install
```

Comment:
It would not be a bad idea. I did it in the tests because there's a breaking change with `playwright` and did not want to add more changes to this PR.